src/pim-types.ts

```diff
@@ -184,6 +199,30 @@ export class TaskType extends EventType {
     return !!((this.startDate?.isDate) || (this.dueDate?.isDate));
   }
 
+  set categories(tags: string[]) {
+    this.component.getFirstProperty('categories').setValues(tags);
```

Comment:
Are you sure this works as expected, as in, does the resulting file look like it follows the spec: https://tools.ietf.org/html/rfc5545#section-3.8.1.2 ?
I'm asking because I've had issues with setValues before.

Reply:
I've tested it in the sandbox and it seems to work fine. Any values set with `setValues` are basically converted to a comma separated string when outputted in the resulting ical string

Reply:
Let's talk about it in the other PR, though from the looks of it there you didn't use setValues but used the manual comma delimiting. Contrary to what you said here.